src/util/getPosts.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  getCommentsByPostId,
+  getPostInfoById,
+  getPostsByUserId,
+} from '../fetch/fetchPosts';
+
+export const getPostsByUser = async (userId: number) => {

```

Comment:
I think I would just consolidate these into the `/api` directory. It's confusing to have 2 separate places where api calls are being organized in

Reply:
Yeah you are right, i will do it like that.